src/utils/link/link-by-prompt.js

```diff
@@ -159,7 +160,7 @@ or run ${chalk.cyanBright('netlify sites:create')} to create a site.`)
 
       let sites
       try {
-        sites = await api.listSites({ filter: 'all' })
+        sites = await listSites({ api, options: { maxPages: 1, filter: 'all' } })
```

Comment:
Limit to 1 page of results since the option here is `Fetching recently updated sites`